Source/WebKit/NetworkProcess/mac/com.apple.WebKit.NetworkProcess.sb.in

```diff
@@ -694,10 +694,11 @@
 (when (and (equal? (param "ENABLE_SANDBOX_MESSAGE_FILTER") "YES") (defined? 'mach-kernel-endpoint))
     (allow mach-kernel-endpoint
         (apply-message-filter
-            (allow mach-message-send (with report) (with telemetry))
+            (deny mach-message-send (with telemetry))
```

Comment:
Ditto: Should we try to get a local report, in case we miss a telemetry window because of other logging?